Gemfile.lock

```diff
@@ -87,7 +93,8 @@ PLATFORMS
 DEPENDENCIES
   cocoapods!
   cocoapods-core!
+  cocoapods-generate!
   xcodeproj!
 
 BUNDLED WITH
-   1.16.1
+   1.17.2
```

Comment:
@paulb777 I'm unsure what this change does, but it was done automatically. Can you especially pay attention to this during review?